samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -204,16 +204,20 @@ private void validateConfig() {
           if (partitions == StreamEdge.PARTITIONS_UNKNOWN) {
             //if the partition is not assigned
             partitions = edgePartitions;
+            log.info("Inferred the partition count {} for the join operator {} from {}."
+                , new Object[] {partitions, join.getOpId(), edge.getName()});
```

Comment:
I don't think we need `new Object[]` here. `log.info()` has an overload that accepts variable-length arguments.

Reply:
My bad. I wasn't looking at the right version of slf4j.